example/src/main/java/com/genius/groupie/example/MainActivity.java

```diff
@@ -70,7 +70,7 @@ protected void onCreate(Bundle savedInstanceState) {
         gray = ContextCompat.getColor(this, R.color.background);
         betweenPadding = getResources().getDimensionPixelSize(R.dimen.padding_small);
 
-        groupAdapter = new GroupAdapter(this);
+        groupAdapter = new GroupAdapter();
```

Comment:
I'd like to keep the possibility of a top level click listener, as this is a pattern many people have continued from ListView.  However, I agree it's a good idea not to require it!  

Can you add `GroupAdapter.setClickListener` as a replacement?